src/components/auth/SignOut.jsx

```diff
@@ -11,4 +14,10 @@ class Logout extends Component {
   }
 }
 
-export default Logout;
+const mapDispatchToProps = dispatch => ({
+  handleLogOut: () => {
+    dispatch(handleLogOut());
```

Comment:
this action should be dispatched on button click, not on component did mount. remove this component at all